server/src/test/java/com/linecorp/centraldogma/server/internal/api/AdministrativeServiceTest.java

```diff
@@ -45,28 +43,19 @@ protected boolean runForEachTest() {
         }
     };
 
-    private static WebClient webClient;
-
-    @BeforeEach
-    void setUp() {
-        final InetSocketAddress serverAddress = dogma.dogma().activePort().get().localAddress();
-        final String serverUri = "http://127.0.0.1:" + serverAddress.getPort();
-        webClient = WebClient.builder(serverUri)
-                             .addHttpHeader(HttpHeaderNames.AUTHORIZATION, "Bearer anonymous")
-                             .build();
-    }
-
     @Test
     void status() {
-        final AggregatedHttpResponse res = webClient.get(API_V1_PATH_PREFIX + "status").aggregate().join();
+        final WebClient client = getClient(dogma);
```

Comment:
Don't we use `dogma.httpClient()` here?
I think we can add `configureHttpClient` to `CentralDogmaExtension`.
```java
protected void configureHttpClient(WebClientBuilder builder) {}
```

Reply:
I'll add this, thanks for the suggestion. But this PR is already pretty big, so if you don't mind, I'd prefer to do this in a follow-up PR.